pkg/vm/evm_prompts.go

```diff
@@ -233,6 +238,10 @@ func PromptSubnetEVMGenesisParams(
 		return SubnetEVMGenesisParams{}, "", err
 	}
 
+	if sc.PoS() || sc.PoA() { // Teleporter bytecode makes genesis too big given the current max size (we include the bytecode for ValidatorManager, a proxy, and proxy admin)
+		params.UseTeleporter = false
```

Comment:
this is not enough, you can still use teleporter, the only thing is that it is not going to be included in genesis
can we keep the teleporter flag on but avoid genesis deploy? but instead deploy afterwards

Reply:
agree. I suggest not fixing it in the scope of this PR and do it as part of the teleporter after blockchain creation deployment PR as we discussed